404.html

```diff
@@ -26,6 +26,23 @@
           btnContainer.append(backBtn);
         }
       }
+      const resp = await fetch(`metadata.json`);
+      if (resp.ok) {
+        const json = await resp.json();
+        if (json && json.data) {
+          const matchedRow = json.data.find((row) => (row.URL === '404.html'));
+          if (matchedRow) {
+            const main = document.querySelector('main');
+            if (main) {
+              main.setAttribute('data-wp-version-name', matchedRow['wp-version-name']);
+              main.setAttribute('data-wp-page-type', matchedRow['wp-page-type']);
+              main.setAttribute('data-wp-http-status', matchedRow['wp-http-status']);
+              main.setAttribute('data-wp-version-traffic', matchedRow['wp-version-traffic']);
+              main.setAttribute('data-wp-page-cookie', matchedRow['wp-page-cookie']);
```

Comment:
This should not be set from metadata but from the integration with the cookie management platform (CMP), as @mhaack pointed out in #12.

Reply:
As discussed with @meryllblanchet, the `main.setAttribute('data-wp-page-cookie',` row will be removed/changed when #12 will be worked.